--- "./KernelSU-Next/kernel/apk_sign - Copy.c"	2024-12-31 16:19:51.886744500 -0500
+++ ./KernelSU-Next/kernel/apk_sign.c	2024-12-31 16:25:51.396912664 -0500
@@ -314,7 +314,13 @@
 
 #endif
 
-bool is_manager_apk(char *path)
+bool ksu_is_manager_apk(char *path)
 {
+#ifdef CONFIG_KSU_SUSFS
+	return (check_v2_signature(path, EXPECTED_NEXT_SIZE, EXPECTED_NEXT_HASH) 
+	|| check_v2_signature(path, 0x3e6, "79e590113c4c4c0c222978e413a5faa801666957b1212a328e46c00c69821bf7") // rsuntk
+	);
+#else
 	return check_v2_signature(path, EXPECTED_NEXT_SIZE, EXPECTED_NEXT_HASH);
+#endif
 }
\ No newline at end of file
